### ISSUE TITLE
How to change the skybox color after 0.15.0?

### ISSUE THREAD

**Issue by pawelngei** [openglobus/openglobus#661]
Since `0.15.0` removed the `backgroundColor` parameter when creating a new `Globe` object, I understand that the proper way would be to use a new `skybox`. I cannot find an example in the documentation. Is there a way to simply set the background color in openglobus > `0.15.0`?

Edit: I dug a little deeper - it seems that the option of `backgroundColor` is still there in `Globe.js`, but it doesn't do anything since the skybox is rendered differently.

**Comment by Zemledelec** [comment]
Yep, the background is changed. Now you can change background colors in the SimpleSkyBackground control:
```
globus.planet.renderer.controls.SimpleSkyBackground.colorOne = "yellow"
globus.planet.renderer.controls.SimpleSkyBackground.colorTwo = "blue"
```

P.S.
...also check `globus.planet.atmosphereEnabled = true` - cool background:)

**Comment by pawelngei** [comment]
Huge thank you! Any chance for putting this in the API docs? :)
